burnman/eos/vinet.py

```diff
@@ -11,10 +11,13 @@
 
 def bulk_modulus(volume, params):
     """
-    compute the bulk modulus as per the third order
-    Vinet equation of state.  Returns bulk
-    modulus in the same units as the reference bulk
-    modulus.  Pressure must be in :math:`[Pa]`.
+    compute the bulk modulus as per the
+    Vinet equation of state.  Reference bulk
+    modulus should be in :math:`[Pa]`.
+    References for this equation of state are :cite:`vinet1986`
+    and :cite:`vinet1987`
+    
+    
```

Comment:
References would be better in the blurb for the Base class (after what is currently line 57)